modules/react/popup/lib/hooks/useTransferOnFullscreenEnter.ts

```diff
@@ -16,7 +16,7 @@ import {usePopupModel} from './usePopupModel';
  */
 export const useTransferOnFullscreenEnter = createElemPropsHook(usePopupModel)(model => {
   const handler = React.useCallback(
-    event => {
+    (event: any) => {
```

Comment:
should these be mouse events instead of any?

Reply:
I need to test it.

Reply:
Neither React events or `screenfull` event don't work correctly. I guess it was a reason why there wasn't assigned type to event before, but it didn't cause type failing. Now not assigned type fails checks, so `any` causes the same skip type checking for `event` argument as it was before.